### PR TITLE
SONARPY-1333 Rule S6559: Fields of a Django ModelForm should be defined explicitly

### PR DESCRIPTION
**Pull request by maksim-grebeniuk-sonarsource**

### CODE REVIEW

File: its/ruling/src/test/java/org/sonar/python/it/RulingHelper.java [hunk 1]
```diff
@@ -143,6 +142,7 @@ static List<String> bugRuleKeys() {
       "S6323",
       "S6328",
       "S6468",
+      "S6559",
```
Comment (guillaume-dequenne-sonarsource) [inline its/ruling/src/test/java/org/sonar/python/it/RulingHelper.java:145]: I admit this is not properly documented (and it's not even very well designed at all) but this `RulingHelper` is meant to list only our bug rules, to be run on the extended ruling.

The rationale was that we thought our existing ruling was not sufficient to ensure proper confidence for bug rules, and yet we didn't want to bloat it and have too many projects on it.

We explicitly decided to only run bug rules on this extended ruling, so maybe it doesn't make sense to add this one here.

This whole approach can be challenged and we may want to do something about it, but for the time being I think we could stay consistent with this approach.

File: python-checks/src/test/resources/checks/django/djangoModelFormFieldsCheck.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from django import forms
+
+class ExcludeFieldsForm(forms.ModelForm):
+    class Meta:
+        model = MyModel
+        exclude = ['field1', 'field2']  # Noncompliant {{Set the fields of this form explicitly instead of using "exclude".}}
```
Comment (guillaume-dequenne-sonarsource) [inline python-checks/src/test/resources/checks/django/djangoModelFormFieldsCheck.py:6]: __Nitpick:__ we could have one assertion on location as well as message, for the sake of completeness.

Comment (maksim-grebeniuk-sonarsource) [reply python-checks/src/test/resources/checks/django/djangoModelFormFieldsCheck.py:6]: done

File: python-checks/src/test/resources/checks/django/djangoModelFormFieldsCheck.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+from django import forms
```
[the remark this answers is not in the document]
Comment (maksim-grebeniuk-sonarsource) [reply python-checks/src/test/resources/checks/django/djangoModelFormFieldsCheck.py:1]: done

File: its/ruling/src/test/java/org/sonar/python/it/RulingHelper.java (hunk 1 above)
Comment (maksim-grebeniuk-sonarsource) [reply its/ruling/src/test/java/org/sonar/python/it/RulingHelper.java]: done

**Pull request merged**